doc/source/methods/CFRL.ipynb

```diff
@@ -14,6 +14,28 @@
     "# Counterfactuals with Reinforcement Learning"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Note\n",
+    "\n",
+    "To enable support for Counterfactuals with Reinforcement Learning, you need one of tensorflow or torch installed. You can do so using:\n",
+    "\n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "or \n",
+    "\n",
+    "```bash\n",
+    "pip install alibi[torch]\n",
+    "```    \n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?